lib/faraday.rb

```diff
@@ -92,6 +92,10 @@ def default_adapter=(adapter)
 
     alias require_lib require_libs
 
+    def respond_to?(symbol)
```

Comment:
Please define [`respond_to_missing?` instead](https://robots.thoughtbot.com/always-define-respond-to-missing-when-overriding). Also, take note that both `respond_to?` and `respond_to_missing?` accept 2 arguments.